### PR TITLE
Fix JWT #152

### PR DESCRIPTION
**Pull request by RossIV**
Fixes/Resolves/Closes/Magic Word's #152 

Change JWT to use `uid` instead of `id` which magically fixed things

### CODE REVIEW

**Comment by kberzinch** [comment]
Fixes #152

File: config/jwt.php [hunk 1]
```diff
@@ -82,10 +82,10 @@
     |
     | Specify a unique property of the user that will be added as the 'sub'
     | claim of the token payload.
-    |
+    |`
```
Comment (kberzinch) [inline config/jwt.php]: ?

Comment (RossIV) [reply config/jwt.php]: uhh..huh.

**Comment by RossIV** [comment]
fixed @kberzinch

**Pull request merged**